test/e2e/common_test.go

```diff
@@ -825,12 +857,19 @@ func (p *PodmanTestIntegration) makeOptions(args []string, noEvents, noCache boo
 		eventsType = "none"
 	}
 
+	networkBackend := p.NetworkBackend.ToString()
+	networkDir := p.NetworkConfigDir
+	if p.NetworkBackend == Netavark {
+		networkDir = p.NetworkConfigDir
+	}
```

Comment:
Same as Dan's comment above; this is tautologous

Reply:
I think you resolved one of the instances but not both??

Anyhow, when you fix this one, wouldn't it be easier to just replace `CNIConfigDir` with `NetworkConfigDir` in line 866 below (and in the other instance)? I don't see a reason for the temporary variable any more.